DashAISoftware/DashAI
# Dataset Summary Step

# Summary


This PR includes the changes to add a new step in the New Dataset flow that allows users to see a summary of the dataset and change the column types by default after de data loading.

## Type of change



- Front end new feature.
- Back end new feature.

## Changes


- Adds the new step in front.
- Implements methods in DashAIDataset class to get and update column types.
- Implements changes on dataset endoint to get and update column types.
- Fix and adds new tests.

## How to Test


To see interface changes:
1. Go to DashAI/ and run ```python -m DashAI```
2. Open a new terminal, go to DashAI/DashAI/front and run ```yarn start```
4. Check the view in Datasets after loading a new dataset.

## Screenshots


![image](https://github.com/DashAISoftware/DashAI/assets/70553082/84bc080d-7e62-46c8-8fe5-2f387fa54b4a)
